okex-price/tests/pricefeed.rs

```diff
@@ -0,0 +1,34 @@
+mod tests {
```

Comment:
no need for `mod tests` here. That is only for unit tests embedded in the files under `src` anything under `tests` is already a test.